### PR TITLE
FIX: LBP API call

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bxx7yl6gka2g236com7wy4mtsi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10478)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: models/fiatUnits.json [hunk 1]
```diff
@@ -1,326 +1,326 @@
 {
     "USD": {
-      "endPointKey": "USD",
-      "symbol": "$",
-      "locale": "en-US",
-      "source": "CoinDesk"
+        "endPointKey": "USD",
+        "locale": "en-US",
+        "source": "CoinDesk",
+        "symbol": "$"
```
Comment (Overtorment) [inline models/fiatUnits.json:6]: changing a bunch of unrelated line is just poor craftsmanship

**Review by Overtorment — CHANGES_REQUESTED** [review]
LGTM KEK

File: models/fiatUnits.json (hunk 1 above)
Comment (marcosrdz) [reply models/fiatUnits.json:6]: Alphabetic sorting

Comment (Overtorment) [reply models/fiatUnits.json:6]: do you want to keep it or it was an accident?

Comment (marcosrdz) [reply models/fiatUnits.json:6]: Keep

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**